droidconKE/chai
# fix/failing-ci-test

---
name: Fix failing ci test
about: fixes failing ci unit tests 
label: bug
Fixes #44 .

---
## How can this PR be tested?
run `make clean test` in the root project directory and watch all unit tests pass

---

Thank you for opening a Pull Request!
Before submitting your PR, there are a few things you can do to make sure it goes smoothly:

Is this your first Pull Request?

Fixes #44 🦕
